Tests/CommonTestTrait.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/*
+ * This file is part of the Sonata project.
+ *
+ * (c) 2018 - Sonata
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SonataHelpers\Tests;
+
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Trait CommonTestTrait
+ *
+ * @author Sylvain Rascar <sylvain.rascar@ekino.com>
+ */
+trait CommonTestTrait
+{
+    /**
+     * @param string $class
+     * @param int    $id
+     *
+     * @return mixed
+     */
+    public function getClassWithId($class, $id)
+    {
+        $item = new $class();
+        $reflection = new \ReflectionClass($item);
+        $reflectionProperty = $reflection->getProperty('id');
+        $reflectionProperty->setAccessible(true);
+        $reflectionProperty->setValue($item, $id);
+
+        return $item;
+    }
+
+    /**
+     * @param string $key
+     * @param array  $params
+     * @param string $domain
+     *
+     * @return string
+     */
+    public function getTranslationString($key, $params, $domain)
```

Comment:
Could be splitted in two traits or classes, with specific responsibilities (one for translation, one for the class instantiation).